src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+public class Node<T> {
```

Comment:
The checklist specifies that `Node<T>` should be an inner class. Consider moving this class inside another class, such as `MyLinkedList`, to adhere to the checklist.